<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaperWindow</class>
 <widget class="QMainWindow" name="PaperWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="refreshBTN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QTableView" name="paperVIEW">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>731</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
